File: sortingAlgorithm/selection_sort/Selectionsort.py
def selectionSort(arr):
    n = len(arr)
    for i in range(0, n):
        min_index = i

        for j in range(i + 1, n):
            if arr[j] < arr[min_index]:
                min_index = j

        arr[min_index], arr[i] = arr[i], arr[min_index]


arr = [69, 99, 23, 55, 11, 45, 66]
selectionSort(arr)
print(arr)

File: sortingAlgorithm/selection_sort/SelectionSort.java
//Time complexity = O(n^2)
class SelectionSort{
	public static void selectionSort(int arr[]){
		int n = arr.length;
		for(int i = 0;i<n;i++){
			int min_index = i;
			for(int j = i+1;j<n;j++){
				if(arr[j]<arr[min_index]){
					min_index = j;
				}
			}
			int temp = arr[min_index];
			arr[min_index] = arr[i];
			arr[i] = temp;
		}
	}
	public static void main(String[] args) {
		int arr[] = {64,89,12,34,56,90,11};
		selectionSort(arr);
		System.out.println("Printint sorted array");
		for(int i = 0;i < arr.length;i++){
			System.out.print(arr[i]+" ");
		}
	}
}
